.travis.yml

```diff
@@ -10,25 +10,18 @@ os:
 # Setting sudo to false opts in to Travis-CI container-based builds.
 sudo: false
 
-# The apt packages below are needed for sphinx builds, which can no longer
-# be installed with sudo apt-get.
-addons:
-    apt:
-        packages:
-            - graphviz
-
 env:
   matrix:
     - PYTHON_VERSION=2.7 EVENT_TYPE='push pull_request cron'
     - PYTHON_VERSION=3.4 SETUPTOOLS_VERSION=20
     - PYTHON_VERSION=3.5
     - PYTHON_VERSION=3.6 SETUPTOOLS_VERSION=dev DEBUG=True
-      CONDA_DEPENDENCIES='sphinx cython numpy packaging six appdirs'
+      CONDA_DEPENDENCIES='sphinx cython numpy packaging six appdirs coverage pytest-cov'
       EVENT_TYPE='push pull_request cron'
 
   global:
-    - CONDA_DEPENDENCIES="setuptools sphinx cython numpy"
-    - PIP_DEPENDENCIES="coveralls pytest-cov"
+    - CONDA_DEPENDENCIES="setuptools sphinx cython numpy coverage pytest-cov"
```

Comment:
It's probably better to install these in ci-helpers when ``--cov`` is stated, isn't it?

Reply:
@bsipocz - yes, though let's do that in a separate PR?

Reply:
yes, and only once it's added to ci-helpers.

Reply:
Hmm, ci-helpers currently install pytest-cov when ``--cov`` is used. I wonder whether coverage is indeed needed or will be installed as the pytest-cov dependency anyway?